scancodeio/settings.py

```diff
@@ -45,7 +45,7 @@
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS", default=[".localhost", "127.0.0.1", "[::1]"])
 
 # SECURITY WARNING: don't run with debug turned on in production
-DEBUG = env.bool("SCANCODEIO_DEBUG", default=False)
+DEBUG = env.bool("SCANCODEIO_DEBUG", default=True)
```

Comment:
This setting is not meant to be changed in that file, put `SCANCODEIO_DEBUG=True` in your env variables instead.

Reply:
Yes, will restore it in the upcomming commit